go.mod

```diff
@@ -70,6 +70,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.14.1
 )
 
+require github.com/go-zookeeper/zk v1.0.3 // indirect
```

Comment:
Nit: can you merge this into the same "require" block below?

Reply:
That was freaking mod tidy. Yeah